### DISCUSSION TITLE
Supporting custom resources that don't have a podSpecTemplate

### DISCUSSION THREAD

**Discussion by shravan-achar** [acrlabs/simkube#135] [Feature Requests and Future Work]
Some custom resources such as Spark Applications (https://github.com/kubeflow/spark-operator/blob/master/config/crd/bases/sparkoperator.k8s.io_sparkapplications.yaml) which do create pods but do not expose a full podSpec template in its custom resource definition. 

Potentially, the tracer can watch pods and track ownership based on labels/annotations instead of the owner reference field.

**Comment by drmorr0** [comment]
Thanks, this is a bug that we should fix; I opened an issue at #137
